enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -4,6 +4,14 @@
 
 The Metabase Embedding SDK for React offers a way to integrate Metabase into your application more seamlessly and with greater flexibility than using the current interactive embedding offering based on iframes.
 
+<div>
+  <a href="https://www.loom.com/share/b6998692937c4ecaab1af097f2123c6f">
+    <img style="max-width: 300px" src="https://cdn.loom.com/sessions/thumbnails/b6998692937c4ecaab1af097f2123c6f-with-play.gif">
+  </a>
+</div>
```

Comment:
This is supported via GitHub's markdown. I'm quite sure that image tags are supported in NPM as well.